### DISCUSSION TITLE
license header for source files

### DISCUSSION THREAD

**Discussion by kvendingoldo** [openbao/openbao#48] [General]
At this moment we have the following header in all source files:

```
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0
```

I suggest to change it to

```
// Copyright (c) HashiCorp, Inc.
// Copyright (c) openbao
// SPDX-License-Identifier: MPL-2.0
```

**Comment by joewxboy** [comment]
Thanks, @kvendingoldo .  I'd want to consult with legal to ensure we're going about it in the proper way.  I'm unsure of the proper attribution for our contributions to files that previously existed in this particular context.

Once the TSC determines which foundation OpenBao falls under, I can find the right legal resource to use.  In the meantime, I can always ask around, unless another member of the community has already asked this question and received an answer.  My goal is to ensure we're not opening our project up to lawsuits due to not following proper rules and conventions.

**Comment by chicks-net** [comment]
I think @kvendingoldo is doing it correctly.  You'll need to decide whether to make a new org to own the copyrights or assign them to the foundation.

* https://opensource.stackexchange.com/questions/11602/what-should-i-do-with-the-copyright-details-in-a-codebase-when-modifying-it-when is one authority which agrees with this approach.
* https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects are broader guidelines from the Linux Foundation and agrees with this approach.
